lib/improver/tests/ensemble_calibration/ensemble_calibration/helper_functions.py

```diff
@@ -53,11 +53,11 @@ def set_up_probability_above_threshold_cube(
     cube_long_name = (
         "probability_of_{}".format(phenomenon_standard_name))
     cube = Cube(data, long_name=cube_long_name,
-                units=phenomenon_units)
+                units=1)
     coord_long_name = "threshold"
     cube.add_dim_coord(
         DimCoord(forecast_thresholds, long_name=coord_long_name,
-                 units='degreesC'), 0)
+                 units=phenomenon_units), 0)
```

Comment:
This change is a good bug fix but you need to change the call to the function everywhere that uses this function. i.e. test_GeneratePercentilesFromProbabilities.py lines 237, 266, 336 and 433 (strangely line 310 is already correct).

Reply:
I've changed these and couldn't find any other cases. These don't affect the unit test results, hence I missed them; good spot.

Reply:
Eek. Changing these calls breaks about half the acceptance tests (or at least a very large fraction of them). Any thoughts on how we proceed?